Tests/test_font_pcf_charsets.py

```diff
@@ -14,38 +15,40 @@
 
 fontname = "Tests/fonts/ter-x20b.pcf"
 
-charsets = {
-    "iso8859-1": {
-        "glyph_count": 223,
-        "message": "hello, world",
-        "image1": "Tests/images/test_draw_pbm_ter_en_target.png",
-    },
-    "iso8859-2": {
-        "glyph_count": 223,
-        "message": "witaj świecie",
-        "image1": "Tests/images/test_draw_pbm_ter_pl_target.png",
-    },
-    "cp1250": {
-        "glyph_count": 250,
-        "message": "witaj świecie",
-        "image1": "Tests/images/test_draw_pbm_ter_pl_target.png",
-    },
+charsets: dict[str, tuple[int, str, str]] = {
+    "iso8859-1": (
+        223,
+        "hello, world",
+        "Tests/images/test_draw_pbm_ter_en_target.png",
+    ),
+    "iso8859-2": (
+        223,
+        "witaj świecie",
+        "Tests/images/test_draw_pbm_ter_pl_target.png",
+    ),
+    "cp1250": (
+        250,
+        "witaj świecie",
+        "Tests/images/test_draw_pbm_ter_pl_target.png",
+    ),
 }
```

Comment:
This original version is clearer: `charsets[encoding]["glyph_count"]` instead of `charsets[encoding][0]`.

Can we keep it? Or use a `NamedTuple`?

https://docs.python.org/3/library/typing.html#typing.NamedTuple

Reply:
Ok, I've pushed a commit with the original dictionary.